### PR TITLE
About and Menu update - Resolves issue #31 copy about SS 

### PR DESCRIPTION
**Pull request by JoacimNilsen**
Added text about SS and how the app works. Also defined texts to typography. Maybe, and maybe after you have a look we might do something about line height.

### CODE REVIEW

**Review by erikpantzar — CHANGES_REQUESTED** [review]
1. Make /about route
2. In the menu, make section 1 sentence "What is SS", and read more links to /about page
3. I think we should work more on the text

**Comment by JoacimNilsen** [comment]
Could use some help with writing a statement or some other solution to make backdrop close upon clicking link. Spent some time on it but decided to move on to next issue!

**Comment by erikpantzar** [comment]
> Could use some help with writing a statement or some other solution to make backdrop close upon clicking link. Spent some time on it but decided to move on to next issue!

Smart, will have lookise

**Comment by JoacimNilsen** [comment]
I fixed the text with alot of <br /> and i dont think the code looks nice with all that. Do you have a suggestion to do it in another way or is it A-OK?

File: src/View/About.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import BackNav from '../Components/BackNav/BackNav'
+
+const About = () => (
+  <div>
+    <BackNav route={{ name: 'Back' }} />
+    <Typography variant="p">
+      Starting Strength is a workout program that gives you the most value for
+      time spent in the gym.
+      <br />
+      <br />
```
Comment (erikpantzar) [inline src/View/About.js]: `<br />` is if you want just one line break. If you want to split text into several different paragraphs, just split them up into separate paragraphs like so:

```
<Typography>
  Text goes here
 </Typography>

<Typography>
    Another pararaph
 </Typography>


<Typography variant="h2">
  I am a heading
</Typography>

<Typography>
  Here I go again being a paragraph. </ br>
  And this is another line that belongs to the same paragraph!
</Typography>

```

**Review by erikpantzar — CHANGES_REQUESTED** [review]

File: src/View/About.js (hunk 1 above)
Comment (JoacimNilsen) [reply src/View/About.js]: Of course! On it!

**Comment by erikpantzar** [comment]
Please push ur commits so I can see them here =) @JoacimNilsen

**Pull request closed without merging**